yarn-project/prover-node/src/prover-node.test.ts

```diff
@@ -425,6 +438,15 @@ describe('prover-node', () => {
           ts: BigInt(Date.now()),
         });
 
+        await retryUntil(
+          async () => {
+            return (await proverNode.getP2P()!.getPeers()).length > 0;
+          },
+          'wait for peers to connect',
+          20,
+          1,
+        );
+
```

Comment:
This is the fix (sometimes the node would broadcast the proof quote _before_ the other peer would connect).

The rest of the diff in this PR is to fix some error in the logs I due to mocking